test/utils/helpers.go

```diff
@@ -257,6 +257,27 @@ func CheckCloudResourceNetworkPolicies(k8sClient client.Client, kind, namespace
 		return v.Status.NetworkPolicyDetails, nil
 	}
 
+	if agent {
+		err := wait.Poll(time.Second*5, time.Second*60, func() (bool, error) {
+			for _, anp := range anps {
+				cmd := fmt.Sprintf("get anp %s -n %s -o json -o=jsonpath={.status.phase}", anp, namespace)
+				out, err := kubeCtl.Cmd(cmd)
+				if err != nil {
+					return false, nil
+				}
+				if strings.Compare(string(out), "Realized") != 0 {
+					logf.Log.V(1).Info("ANP realization in progress", "IDS", ids, "ANP", anp)
+					return false, nil
+				}
+			}
+			return true, nil
+		})
+		if err != nil {
+			logf.Log.V(1).Info("ANP realization failed")
+		}
+		return err
+	}
+
```

Comment:
we would also need to collect logs from VM.
Update function CollectSupportBundle to collect Vm logs from /var/log/antrea-agent/